js/bybit.js

```diff
@@ -3503,19 +3397,131 @@ module.exports = class bybit extends Exchange {
          * @returns {object} an [order structure]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
          */
         await this.loadMarkets ();
+        this.checkRequiredSymbol ('createOrder', symbol);
         const market = this.market (symbol);
         symbol = market['symbol'];
-        const enableUnifiedMargin = await this.isUnifiedMarginEnabled ();
+        const [ enableUnifiedMargin, enableUnifiedAccount ] = await this.isUnifiedEnabled ();
         const isUSDCSettled = market['settle'] === 'USDC';
+        if (enableUnifiedAccount && !market['inverse']) {
+            return await this.createUnifiedAccountOrder (symbol, type, side, amount, price, params);
+        } else if (market['spot']) {
+            return await this.createSpotOrder (symbol, type, side, amount, price, params);
+        } else if (enableUnifiedMargin && !market['inverse']) {
+            return await this.createUnifiedMarginOrder (symbol, type, side, amount, price, params);
+        } else if (isUSDCSettled) {
+            return await this.createUsdcOrder (symbol, type, side, amount, price, params);
+        } else {
+            return await this.createContractV3Order (symbol, type, side, amount, price, params);
+        }
+    }
+
+    async createUnifiedAccountOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        if (price === undefined && type === 'limit') {
```

Comment:
@sc0Vu the user can provide **L**imit and **M**arket so you should probably use the lowercase type in this condition